### ISSUE TITLE
NTR: IgM anti-polio antibodies (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8706]
New term request IgM anti-polio antibodies (segal_070425210131): immunoglobulins, antibodies

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000346
- label: Anti-polio virus antibody positivity
- parents: [Positive bloodstream virus-specific antibody positivity (HP:0430059)](https://hpo.jax.org/app/browse/term/HP:0430059)
- pmid: PMID:32644370
- def: The presence of antibodies in the blood circulation that react against a component of poliovirus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8706
- synonyms: 

Note we will be addiing modifiers for IgG, IgG etc
